### PR TITLE
taiga task #303 - first rules for cloudreadieness

### PR DESCRIPTION
**Pull request by mareknovotny**

### CODE REVIEW

File: rules-reviewed/cloud/java-rmi.windup.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        <description>
+            This is a ruleset for Java Remote Method Invocation specific rules while migrating to Cloud environment.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.7.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.7.0.Final" />
+        </dependencies>
+        <sourceTechnology id="java" />
+        <sourceTechnology id="rmi" />
+        <targetTechnology id="cloud" />
```
Comment (jsight) [inline rules-reviewed/cloud/java-rmi.windup.xml]: Perhaps we should have a more descriptive name than "cloud"?

Comment (mareknovotny) [inline rules-reviewed/cloud/java-rmi.windup.xml]: :) perhaps yes, but I do not get idea what could be more suitable one-word tag

Comment (mareknovotny) [inline rules-reviewed/cloud/java-rmi.windup.xml]: or technology ;)

Comment (jsight) [inline rules-reviewed/cloud/java-rmi.windup.xml]: @d-s - Any opinions?

Comment (mareknovotny) [inline rules-reviewed/cloud/java-rmi.windup.xml]: I could imagine only saas - software as a service, but it is too long and shortcuts are not clear to everybody

Comment (jsight) [inline rules-reviewed/cloud/java-rmi.windup.xml]: Really, I would prefer "openshift" or some other product name. The "cloud" will have different rules depending upon which cloud offering is used.

**Comment by mareknovotny** [comment]
I re-phrased hint suggestions too.

**Comment by d-s** [comment]
The hint suggestions look the same as far as I can tell (other than s/cloud/openshift/). Can you just remove the text from the description for now? When I try out this ruleset on a couple apps I can add in some temporary description text for us to use until we have documents to point to.

**Comment by mareknovotny** [comment]
@d-s I can't leave out messages completely due tests and our own validity of rules we set. Please review this changes, which are more re-phrasing.

**Comment by jsight** [comment]
@d-s Were you able to look over the latest version of this?

**Comment by d-s** [comment]
Sorry I didn't get to this before. I took a shot at writing up some hint messages for these rules and sent it as a PR to Marek's branch.

https://github.com/mareknovotny/windup-rulesets/pull/3
@jsight @mareknovotny PTAL

**Comment by mareknovotny** [comment]
@d-s that's fine I got a notification about https://github.com/mareknovotny/windup-rulesets/pull/3
I will take what I can from that, but some messages are the same and there needs to be a differentiation there otherwise the rules should be merged into one rule.

**Comment by d-s** [comment]
OK, we can add a first line to the message for each rule that relates to
the title and is different from the other hints' messages.

Later when we have KB articles for these topics we can update the hints
with more specific messages.

**Comment by mareknovotny** [comment]
agreed

**Comment by mareknovotny** [comment]
I expect this PR update tests will not pass as windup core master is broken right now

**Comment by mareknovotny** [comment]
retest

**Pull request merged**